client/gutenberg/extensions/shared/frontend-management.js

```diff
@@ -15,20 +15,22 @@ export class FrontendManagement {
 	initializeFrontendReactBlocks( component, options = {}, rootNode ) {
 		const { name, attributes } = options.settings;
 		const { selector } = options;
-		const blockClass = [ '.wp-block', name.replace( '/', '-' ) ].join( '-' );
-		rootNode.querySelectorAll( blockClass ).forEach( node => {
-			const data = this.extractAttributesFromContainer( node.dataset, attributes );
+		const blockClass = `.wp-block-${ name.replace( '/', '-' ) }`;
+		const blockNodes = Array.prototype.slice.apply( rootNode.querySelectorAll( blockClass ) );
+		blockNodes.forEach( node => {
```

Comment:
For the record: [MDN says `NodeList.prototype.forEach` is not supported in IE.](https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach#Browser_Compatibility)

---

Why not just use a `for` loop?

```js
for (var i = 0; i < myNodeList.length; i++) {
  var item = myNodeList[i];
}
```